app/scripts/modules/core/pipeline/config/stages/createLoadBalancer/createLoadBalancerExecutionDetails.html

```diff
@@ -0,0 +1,60 @@
+<div ng-controller="createLoadBalancerExecutionDetailsCtrl">
+    <execution-details-section-nav sections="configSections"></execution-details-section-nav>
+    <div class="step-section-details" ng-if="detailsSection === 'loadBalancerConfig'">
+        <div class="row">
+
```

Comment:
Yes. It aligns well after adding <div class="col-md-12">

![image](https://cloud.githubusercontent.com/assets/16142228/13861275/979f867a-ec49-11e5-860b-7ed8b54d2c6f.png)